frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -888,12 +888,25 @@
         };
 
         vm.changeSubmissionVisibility = function(submission_id) {
+            var dateTime = new Date();
+            dateTime = dateTime.toISOString();
+            var splitDateTime = dateTime.split("T");
+            var date = splitDateTime[0];
+            var time = splitDateTime[1].split(".")[0];
```

Comment:
The datetime should be computed at the backend and not at the frontend. Both are decouped.